vms/platformvm/utxo/handler.go

```diff
@@ -485,9 +491,17 @@ func (h *handler) VerifySpend(
 			)
 		}
 		utxos[index] = utxo
+		signers[index], err = utxoDB.GetMultisigUTXOSigners(utxo)
+		if err != nil {
+			return fmt.Errorf(
+				"failed to get multisig info consumed UTXO %s due to: %w",
+				&input.UTXOID,
+				err,
+			)
+		}
 	}
 
-	return h.VerifySpendUTXOs(tx, utxos, ins, outs, creds, unlockedProduced)
+	return h.VerifySpendUTXOs(tx, utxos, ins, outs, creds, signers, unlockedProduced)
 }
 
 func (h *handler) VerifySpendUTXOs(
```

Comment:
With this approach you also need to modify VerifyLockUTXOs and VerifyUnlockDepositedUTXOs, cause they also do sigs check